### PR TITLE
Log in new

### PR DESCRIPTION
**Pull request by thalice**

### CODE REVIEW

File: frontend/src/components/LogInForm.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <input
+          className="logInInput"
+          type="text"
+          value={this.state.password}
+          onChange={event => {
+            this.setState({ password: event.target.value });
+          }}
+        />
+        <p className="signUp">
+          <NavLink to="/signInForm">
```
Comment (laurentmahieu) [inline frontend/src/components/LogInForm.jsx]: why a "NavLink" and not a "Link"?

File: frontend/src/components/TopBar.jsx [hunk 2]
```diff
@@ -24,13 +25,14 @@ class TopBar extends React.Component {
           <div className="topBarNavigationItems">
             <ul>
               <li>
-                <NavLink activeClassName="active" to="/">
+                <Modal />
+                {/* <NavLink activeClassName="active" to="/">
```
Comment (laurentmahieu) [inline frontend/src/components/TopBar.jsx]: death code ?

File: frontend/src/components/LogInForm.jsx (hunk 1 above)
Comment (thalice) [reply frontend/src/components/LogInForm.jsx]: Changed,my Captain'!

File: frontend/src/components/TopBar.jsx (hunk 2 above)
Comment (thalice) [reply frontend/src/components/TopBar.jsx]: Dead and buried now, my Cap'tain!

Comment (LoicBrassart) [reply frontend/src/components/TopBar.jsx]: Ohnoz, that's undead code, it came back! :scream:

File: frontend/src/components/LogInForm.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                  
+        <input
+          className="logInInput"
+          type="text"
+          value={this.state.username}
+          onChange={event => {
+            this.setState({ username: event.target.value });
+          }}
+        />
+             <label className="label">Password</label>
```
Comment (LoicBrassart) [inline frontend/src/components/LogInForm.jsx]: Indentation fluke

Comment (thalice) [reply frontend/src/components/LogInForm.jsx]: Halloween is over, but still (un)dead code made its way through...should be definitely dead now. 
For the indentation, "Prettier" keeps saving my indentation this way, so I can't correct it without being corrected...Computer 1- Human 0 :(

**Pull request merged**